common/message.cpp

```diff
@@ -175,6 +175,45 @@ Message Message::readMessage(QIODevice *device)
     return msg;
 }
 
+quint8 Message::lowestSupportedDataVersion()
+{
+    return QDataStream::Qt_4_8;
+}
+
+quint8 Message::highestSupportedDataVersion()
+{
+#if QT_VERSION >= QT_VERSION_CHECK(5, 0, 0)
```

Comment:
I think we can just return `Qt_DefaultCompiledVersion` here.

From qdatastream.h:
```
#if QT_VERSION >= 0x050900
#error Add the datastream version for this Qt version and update Qt_DefaultCompiledVersion
#endif
        Qt_DefaultCompiledVersion = Qt_5_8
```